.changeset/calm-yaks-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@guardian/google-admanager-api": minor
```

Comment:
Type alterations are potentially breaking changes so this may need to be a major patch- I'm not 100% on this one as the types that have changed weren't previously directly part of the public interface, but these changes may affect code that consumes the creative template service.

Generally speaking a minor version change is fully backwards-compatible and can be updated to safely without making changes to consuming code- see https://semver.org/

Reply:
Agreed this would be a major bump, I will amend 👍